doc/modules/calibration.rst

```diff
@@ -205,3 +205,115 @@ a similar decrease in log-loss.
+using the midpoint of the space of possible values. Estimating a decision threshold
+for a specific use case can help to increase the overall accuracy of the model and
+provide better handling for sensitive classes.
+
+.. currentmodule:: sklearn.calibration
+
+:class:`CutoffClassifier` can be used as a wrapper around a model for binary
+classification to help obtain a more appropriate decision threshold and use it
+for predicting new samples.
+
+Usage
+-----
+
+To use the :class:`CutoffClassifier` you need to provide an estimator that has
+a ``decision_function`` or a ``predict_proba`` method. The ``strategy`` parameter
+controls whether the first will be preferred over the second if both are available.
+
+The wrapped estimator can be pre-trained, in which case ``cv = 'prefit'``, or not. If
+the classifier is not trained then a cross-validation loop specified by the parameter
+``cv`` can be used to obtain a decision threshold by averaging all decision thresholds
+calculated on the hold-out parts of each cross validation iteration. Finally the model
+is trained on all the provided data. When using ``cv = 'prefit'`` you need to make sure
+to use a hold-out part of your data for calibration.
+
+The methods for finding appropriate decision thresholds are based either on precision
+recall estimates or true positive and true negative rates. Specifically:
+
+.. currentmodule:: sklearn.metrics
+
+* ``f_beta``
+   selects a decision threshold that maximizes the :func:`fbeta_score`. The value of
+   beta is specified by the parameter ``beta``. The ``beta`` parameter determines
+   the weight of precision. When ``beta = 1`` both precision recall get the same
+   weight therefore the maximization target in this case is the :func:`f1_score`.
+   if ``beta < 1`` more weight is given to precision whereas if ``beta > 1`` more
+   weight is given to recall.
+
+* ``roc``
+   selects the decision threshold for the point on the :func:`roc_curve` that is
+   closest to the ideal corner (0, 1)
+
+* ``max_tpr``
+   selects the decision threshold for the point that yields the highest true
+   positive rate while maintaining a minimum true negative rate, specified by
+   the parameter ``threshold``
+
+* ``max_tnr``
+   selects the decision threshold for the point that yields the highest true
+   negative rate while maintaining a minimum true positive rate, specified by
+   the parameter ``threshold``
+
+Here is a simple usage example::
+
+   >>> from sklearn.calibration import CutoffClassifier
+   >>> from sklearn.datasets import load_breast_cancer
+   >>> from sklearn.naive_bayes import GaussianNB
+   >>> from sklearn.model_selection import train_test_split
+   >>>
+   >>> X, y = load_breast_cancer(return_X_y=True)
+   >>> X_train, X_test, y_train, y_test = train_test_split(
+   ...     X, y, train_size=0.6, random_state=42)
+   >>> clf = CutoffClassifier(GaussianNB(), method='roc', cv=3).fit(
+   ...     X_train, y_train)
```

Comment:
You should at least call predict now or else you're not really showing how to use it

Reply:
true that